### ISSUE TITLE
Proposal peer review - from Howell-Xia

### ISSUE THREAD

**Issue by michaelinwords** [INFO-523-SU25/final-project-Herling-Mi#1]
The following is the peer review of the project proposal by Howell-Xia. The team members that participated in this review are:

- Cat Xia - @CatXia810
- (Ahn) Michael Howell - @michaelinwords

- _Describe the goal of the project._
The team is planning to build an AI-driven music recommendation system with machine learning applied to audio data. They intend to build a model that recognizes the language in each song and combine it with genre classification.

- _Describe the data used or collected._
The team is using audio data sourced from Youtube through retrieving the top 5 songs of a manually curated artist list. They intend to create 4 custom scripts to do this and will store the data in a JSON file.

- _Describe how the research question will be answered, e.g. what approaches / methods will be used._
They will be using sklearn, lightgbm, xgboost libraries for machine learning. They will be using torch, tensorflow for deep learning. They will be using librosa, torchaudio, and Demucs for audio processing. They plan to process each audio file as .mp3 or .wav. They intend to use K-means clustering or Hierarchical clustering to identify similar groups and train this data through cross-validation and hyperparameter tuning with grid or random search. The model will be evaluated on precision, recall, accuracy, and f1-score. Ablation Analysis will be used to access the impact of vocal or non-vocal feature inclusion.

- _Is there anything that is unclear from the proposal?_
I thought the proposal was very clear and organized. I was wondering about the Likart score and if that’s the target variable that the model will be predicting?

- _Provide constructive feedback on how the team might be able to improve their project._
Would that necessarily mean that each person the model is predicting for would need to have a different Likart score database?

- _What aspect of this project are you most interested in and would like to see highlighted in the presentation._
(Cat) I’m really looking forward to the machine learning and deep learning aspects of this project! (Ahn) I'm curious to see how your approach to the language (lyric) interpretation/NLU side develops, especially given sung speech is produced with some notable differences from usual speech. Also (potentially out of scope) I'm curious to learn more about how Demucs isolates vocals based on acoustic features.

- _Provide constructive feedback on any issues with file and/or code organization._
The code looks good and is well organized and explained.

- _(Optional) Any further comments or feedback?_
I believe you may need to publish the proposal to the website and link it to your Github homepage.
